.github/workflows/run-acceptance-tests.yml

```diff
@@ -421,6 +421,12 @@ jobs:
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
         version: v2.5.0
+    - name: make upstream
```

Comment:
it'd be great if we can put this in ci-mgmt, instead of hacking it in every provider. Out of scope here.